controllers/storageclassclaim/storageclassclaim_controller.go

```diff
@@ -410,6 +388,8 @@ func (r *StorageClassClaimReconciler) reconcileProviderPhases() (reconcile.Resul
 			r.storageClassClaim.Status.Phase = v1alpha1.StorageClassClaimReady
 		}
 
+	} else {
+		r.storageClassClaim.Status.Phase = v1alpha1.StorageClassClaimStateDeleting
```

Comment:
Why are we removing the indication that we are at a deletion phase? 
I think this is important information to convey as deletion is not immediate and the claim will stay alive until all sub-resources are deleted.

Reply:
I'm updating here that we are at the deletion stage, until I'm missing something here